### ISSUE TITLE
Datasets?

### ISSUE THREAD

**Issue by IamMI** [facebookresearch/DiT#110]
I’m currently trying to locate the training datasets. Although the paper specifies ImageNet, after downloading [ILSVRC 2012](https://image-net.org/challenges/LSVRC/2012/index.php), I noticed that the labels don’t align with those in the code!

For instance, in the code, I set class_labels=[15] to generate an image of a bird, but in the ILSVRC 2012 dataset, I found a dataset containing images of a sea lion instead!

This has piqued my curiosity about the exact dataset used to train the model.
